adapters/all/opentelemetry-adapters-faraday.gemspec

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+# Copyright 2020 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+lib = File.expand_path('lib', __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+
+require 'opentelemetry/adapters/all/version'
+version = OpenTelemetry::Adapters::All::VERSION
+
+Gem::Specification.new do |spec|
+  spec.name        = 'opentelemetry-adapters-all'
+  spec.version     = OpenTelemetry::Adapters::All::VERSION
+  spec.authors     = ['OpenTelemetry Authors']
+  spec.email       = ['cncf-opentelemetry-contributors@lists.cncf.io']
+
+  spec.summary     = 'All-in-one instrumentation adapter bundle for the OpenTelemetry framework'
+  spec.description = 'All-in-one instrumentation adapter bundle for the OpenTelemetry framework'
+  spec.homepage    = 'https://github.com/open-telemetry/opentelemetry-ruby'
+  spec.license     = 'Apache-2.0'
+
+  spec.files = ::Dir.glob('lib/**/*.rb') +
+               ::Dir.glob('*.md') +
+               ['LICENSE']
+  spec.require_paths = ['lib']
+  spec.required_ruby_version = '>= 2.4.0'
+
+  spec.add_dependency 'opentelemetry-adapters-concurrent_ruby', version
+  spec.add_dependency 'opentelemetry-adapters-ethon', version
+  spec.add_dependency 'opentelemetry-adapters-excon', version
+  spec.add_dependency 'opentelemetry-adapters-faraday', version
+  spec.add_dependency 'opentelemetry-adapters-net_http', version
+  spec.add_dependency 'opentelemetry-adapters-rack', version
+  spec.add_dependency 'opentelemetry-adapters-redis', version
+  spec.add_dependency 'opentelemetry-adapters-restclient', version
+  spec.add_dependency 'opentelemetry-adapters-sinatra', version
```

Comment:
This is going to require the versions of all adapters to be kept in sync. This is fine for now, but probably won’t be in the future. It might be useful to open an issue to remind us of this.

Likewise, it’d be helpful to document this for release management, so we remember to bump versions and release all the adapters as part of the general release process. If we *want* to coordinate versions of all the gems, we should automate the version bumps, but I don’t think that’s the right thing to do in the long term.

Reply:
Right now I am keeping to strict versioning to allow for changes in the API. As things stabilize, I agree, we'll want to relax this policy. I'll update the release docs with the process for releasing this gem and get an issue started to note and track expected changes to it.